Competition2017/src/competition/subsystems/agitator/AgitatorSubsystem.java

```diff
@@ -30,7 +32,7 @@
     private final PIDPropertyManager pidProperties;
 
     public AgitatorSubsystem(int motor, RobotSide side, WPIFactory factory, XPropertyManager propManager, RobotAssertionManager assertionManager){
-        log.info("Creating " + side + " Agitator");
```

Comment:
In order for it to work removing the 'side' here we'll need to update the name to reflect that instead so for these places we need a call like:
`super(side + " Agitator");`